.github/workflows/docs.yml

```diff
@@ -39,12 +64,18 @@ jobs:
           mkdocs build
 
       - name: Deploy docs
-        if: ${{ github.event_name == 'push' || ! github.event.pull_request.head.repo.fork }}
+        # Run only if push is to `main`, or if it's a PR not from a fork.
+        if: ${{ (github.event_name == 'push' && github.ref == 'refs/heads/main') || (github.event_name == 'pull_requests' && ! github.event.pull_request.head.repo.fork) }}
```

Comment:
Mumble mumble, this isn't working as expected, [deployment was skipped on this PR](https://github.com/ukri-excalibur/excalibur-tests/actions/runs/8361574484/job/22889981483?pr=289) :thinking: